python/tritonserver/_api/_response.py

```diff
@@ -308,9 +304,6 @@ def _response_callback(self, response, flags, unused):
             self._queue.put(response)
             if self._user_queue is not None:
                 self._user_queue.put(response)
-            if flags == TRITONSERVER_ResponseCompleteFlag.FINAL:
-                del self._request
```

Comment:
Note to self to verify that this might fix the issue with `server.stop()` in `L0_python_api`.

Reply:
Passed 10/10 reruns, so hoping it's the fix. We'll see for sure later on.